apps/src/templates/sectionProgress/progressTables/progress-table-constants.module.scss

```diff
@@ -6,11 +6,13 @@ $row-height: 42px;
 $expanded-row-height: 30px;
 $max-rows: 14;
 $max-height: $max-rows * $row-height + $scrollbar-height;
+$table-width: calc($content-width - 64px);
```

Comment:
This includes the padding added in teacher dashboard [here](https://github.com/code-dot-org/code-dot-org/blob/855a06fec450c3bc0ba1487efaf71759f735c157/apps/src/templates/teacherDashboard/teacher-dashboard.module.scss)